webapp/channels/src/components/access_problem/index.tsx

```diff
@@ -1,20 +1,44 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {useEffect} from 'react';
+import React, {useCallback, useEffect} from 'react';
 import {useIntl} from 'react-intl';
+import {useHistory} from 'react-router-dom';
 
 import {trackEvent} from 'actions/telemetry_actions';
 
 import AccessProblemSVG from 'components/common/svg_images_components/access_problem_svg';
 
 import './access_problem.scss';
+import type {CustomizeHeaderType} from '../header_footer_route/header_footer_route';
 
-const AccessProblem = () => {
+type AccessProblemProps = {
+    onCustomizeHeader?: CustomizeHeaderType;
+}
+
+const AccessProblem = ({
+    onCustomizeHeader,
+}: AccessProblemProps) => {
     const {formatMessage} = useIntl();
+    const history = useHistory();
+
+    const handleHeaderBackButtonOnClick = useCallback(() => {
+        trackEvent('signup', 'access_problem__click_back');
```

Comment:
The changes are inspired by the the signup page (`signup.tsx`). There, the back button handler looks like this:
```
    const handleHeaderBackButtonOnClick = useCallback(() => {
        if (!noAccounts) {
            trackEvent('signup_email', 'click_back');
        }

        history.goBack();
    }, [noAccounts, history]);
```

I don't fully understand why it should only track and event when `noAccounts` is falsy. Possibly, we should apply the same logic here?
I also don't know the conventions for the names of tracking events. This might have to be tweaked.